changelogs/unreleased/5940-izturn-minor.md

```diff
@@ -0,0 +1,4 @@
+## HTTPRoute Timeouts
```

Comment:
Please add a link to https://gateway-api.sigs.k8s.io/reference/spec/#gateway.networking.k8s.io/v1.HTTPRouteTimeouts somewhere in the changelog, as a reference for readers.